### PR TITLE
Adds more cleric spells and more

### PR DESCRIPTION
**Pull request by Alberyk**
-adds the mend would spell; a spell that allows the wizard to close bleeding, heal internal bleeding, broken bones and organ damage by selecting the hurt zone
-adds the resurrection spell, that brings a dead target to life, it will not work if he has been dead for too long
-adds mind bend, that allows the wizard to turn his victim into his lobotomized slave, like the vampire thrall power
-buffs some healing spell, summon bats and summon bear
-adds more wizard related sounds
-adds an emote when someone is firing their eye lasers

### CODE REVIEW

File: code/modules/spells/aoe_turf/conjure/druidic_spells.dm [hunk 1]
```diff
@@ -21,16 +21,14 @@
...
+	charge_max = 300
 	invocation = "Bla'yo daya!"
 	invocation_type = SpI_SHOUT
 	level_max = list(Sp_TOTAL = 3, Sp_SPEED = 3, Sp_POWER = 3)
-	cooldown_min = 600
 
 	range = 1
 
-	summon_amt = 1
+	summon_amt = 4
```
Comment (Arrow768) [inline code/modules/spells/aoe_turf/conjure/druidic_spells.dm]: If I understand that correctly, thats 16 times the amount as before ?!

File: code/modules/spells/targeted/cleric_spells.dm [hunk 2]
```diff
@@ -110,3 +111,144 @@
 
 
 	return "You will now heal twice as much, but take twice as much damage. It will probably kill you."
+
+/spell/targeted/mend
+	name = "Mend Wounds"
+	desc = "This spell heals internal wounds and broken bones."
+	feedback = "MW"
+	spell_flags = INCLUDEUSER | SELECTABLE | NEEDSCLOTHES
+	invocation = "Ges'undh'eit"
```
Comment (Arrow768) [inline code/modules/spells/targeted/cleric_spells.dm]: Thats the german word for health.
Gesundheit

File: code/modules/mob/living/simple_animal/hostile/commanded/bear_companion.dm [hunk 3]
```diff
@@ -7,19 +7,21 @@
...
+	health = 100
+	maxHealth = 100
 
 	density = 1
 
 	attacktext = "swatted"
-	melee_damage_lower = 10
-	melee_damage_upper = 10
+	melee_damage_lower = 25
+	melee_damage_upper = 25
```
[the remark this answers is not in the document]
Comment (Alberyk) [reply code/modules/mob/living/simple_animal/hostile/commanded/bear_companion.dm]: Yes, they are commanded animals and the spell itself has a really long cooldown.

File: code/modules/spells/aoe_turf/conjure/druidic_spells.dm (hunk 1 above)
Comment (Alberyk) [reply code/modules/spells/aoe_turf/conjure/druidic_spells.dm]: Bats are really weak, it takes one or two hits to kill them.

File: code/modules/spells/targeted/cleric_spells.dm (hunk 2 above)
Comment (Alberyk) [reply code/modules/spells/targeted/cleric_spells.dm]: That is on puporse, a lot of wizard's invocations are references or 'jokes".

**Pull request merged**